src/System.Management.Automation/security/SecurityManager.cs

```diff
@@ -184,6 +180,11 @@ private bool CheckPolicy(ExternalScriptInfo script, PSHost host, out Exception r
                 }
             }
 
+            // WLDP and Applocker takes priority over powershell exeuction policy.
+            // See if they want to bypass the authorization manager
+            if (executionPolicy == ExecutionPolicy.Bypass)
```

Comment:
Nitpick.  I believe our style guidelines say the conditional statement must be within brackets.
```suggestion
            if (executionPolicy == ExecutionPolicy.Bypass)
            {
                return true;
            }
```